src/test/java/seedu/application/logic/parser/ParserUtilTest.java

```diff
@@ -86,7 +92,93 @@ public void parseCompany_validValueWithWhitespace_returnsTrimmedCompany() throws
     }
 
     @Test
-    public void parseIndustry_invalidValue_throwsParseException() throws Exception {
+    public void parseDeadline_null_throwsNullPointerException() {
```

Comment:
Good splitting of EP for parse method & identification of BVA like with & without whitespaces